lrauv_ignition_plugins/CMakeLists.txt

```diff
@@ -254,11 +254,15 @@ configure_file(
 #============================================================================
 # World generation
 set(WORLD_NAME "portuguese_ledge")
+set(WORLD_NAME_EMPTY "empty_environment_with_tiles")
 add_custom_command(
   OUTPUT world_gen_cmd
   COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/worlds/empy_expander.py
   ${CMAKE_CURRENT_SOURCE_DIR}/worlds/${WORLD_NAME}.sdf.em
   ${CMAKE_CURRENT_BINARY_DIR}/worlds/${WORLD_NAME}.sdf
+  COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/worlds/empy_expander.py
+  ${CMAKE_CURRENT_SOURCE_DIR}/worlds/${WORLD_NAME_EMPTY}.sdf.em
+  ${CMAKE_CURRENT_BINARY_DIR}/worlds/${WORLD_NAME_EMPTY}.sdf
```

Comment:
@adityapande-1995 meta: at some point, it would be nice to have a macro to set up SDF template expansion during builds.